pkg/interface/chat/src/js/components/lib/chat-input.js

```diff
@@ -329,4 +198,29 @@ export class ChatInput extends Component {
       </div>
     );
   }
+
+  render() {
+    const { props, state } = this;
+
+    let writePermission = props.permissions[`/chat${props.station}/write`];
```

Comment:
Just a minor note: it might be error-prone to include the `/` prefix in `props.station`.